### PR TITLE
feat: ssh embedded server for container access

### PR DESCRIPTION
**Pull request by antmat**
Early version without authentication.
@3Hren @noxiouz @Sokel @sshaman1101 
PTAL

### CODE REVIEW

**Comment by Sokel** [comment]
good;
shh will be always on?

**Comment by antmat** [comment]
We can make it configurable, if we decide to.

**Comment by Sokel** [comment]
I think, it will be configurable by flag -noshh or any

**Comment by Sokel** [comment]
About authentication: would you research it with *ecdsa.Private key?

**Comment by noxiouz** [comment]
@Sokel https://godoc.org/golang.org/x/crypto/ssh#NewPublicKey
It should work out of the box

File: insonmnia/miner/builder.go [hunk 1]
```diff
@@ -53,6 +54,10 @@ func (b *MinerBuilder) UUID(uuid string) *MinerBuilder {
 	return b
 }
 
+func (b *MinerBuilder) SSH(ssh SSH) {
+	b.ssh = ssh
```
Comment (noxiouz) [inline insonmnia/miner/builder.go]: return *MinerBuilder?

File: insonmnia/miner/ssh.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+		outputErr <- err
+	}()
+
+	go func() {
+		defer stream.CloseWrite()
+		_, err := io.Copy(stream.Conn, session)
+		outputErr <- err
+	}()
+
+	err = <-outputErr
```
Comment (noxiouz) [inline insonmnia/miner/ssh.go]: goroutine leak: you write 2 errors in the chan, but read only 1

**Pull request merged**